react-design/react-design-notes.md

```diff
@@ -0,0 +1,48 @@
+# react-design-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What are the phases in the React design process?
```

Comment:
This question is referring to the three phases we walked through in the exercise.